server/routes/serviceProviderReferrals/showReferralPresenter.ts

```diff
@@ -17,30 +17,67 @@ export default class ShowReferralPresenter {
     title: `${utils.convertToProperCase(this.serviceCategory.name)} referral for ${ReferralDataPresenterUtils.fullName(
       this.referralFields.serviceUser
     )}`,
+    interventionDetailsSummaryHeading: `${utils.convertToProperCase(this.serviceCategory.name)} intervention details`,
   }
 
   readonly probationPractitionerDetails: SummaryListItem[] = [
     { key: 'Name', lines: [`${this.sentBy.firstName} ${this.sentBy.surname}`], isList: false },
     { key: 'Email address', lines: [this.sentBy.email ?? ''], isList: false },
   ]
 
+  get interventionDetails(): SummaryListItem[] {
+    const selectedDesiredOutcomes = this.serviceCategory.desiredOutcomes
+      .filter(desiredOutcome => this.referralFields.desiredOutcomesIds.includes(desiredOutcome.id))
+      .map(desiredOutcome => desiredOutcome.description)
+
+    const selectedComplexityLevel = this.serviceCategory.complexityLevels.find(
+      complexityLevel => complexityLevel.id === this.referralFields.complexityLevelId
+    )
+
+    const complexityLevelText = {
+      level: selectedComplexityLevel?.title || 'Level not found',
+      text: selectedComplexityLevel?.description || 'Description not found',
+    }
+
+    return [
+      { key: 'Sentence information', lines: ['Not currently set'], isList: false },
+      { key: 'Desired outcomes', lines: selectedDesiredOutcomes, isList: true },
+      { key: 'Complexity level', lines: [complexityLevelText.level, complexityLevelText.text], isList: false },
+      {
+        key: 'Date to be completed by',
+        lines: [ShowReferralPresenter.govukFormattedDateFromStringOrNull(this.referralFields.completionDeadline)],
+        isList: false,
+      },
+      {
+        key: 'Maximum number of enforceable days',
+        lines: [this.referralFields.usingRarDays ? String(this.referralFields.maximumRarDays) : 'N/A'],
+        isList: false,
+      },
+      {
+        key: 'Further information for the provider',
+        lines: [this.referralFields.furtherInformation || 'N/A'],
+        isList: false,
+      },
+    ]
+  }
+
   readonly serviceUserNotificationBannerArgs = {
     titleText: 'Service user details',
     html:
       `<p class="govuk-notification-banner__heading">${this.serviceUser.firstName} ${this.serviceUser.surname}<p>` +
-      `<p>Date of birth: ${this.serviceUserDateOfBirth}</p>` +
+      `<p>Date of birth: ${ShowReferralPresenter.govukFormattedDateFromStringOrNull(
```

Comment:
do we want to display something is this isn't present? like "dob unknown" or something?

Reply:
nvm, seen output below